src/Entity/Tags.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use ApiPlatform\Metadata\ApiResource;
+
+#[ORM\Entity]
+#[ApiResource]
+class Tags
```

Comment:
It's a single tag or multiple tags ?

Reply:
It's a single Tag but for the update i can't remove/change thhe name to fiield Tag

Reply:
A single Tag, it's a error . i need to change that.